### DISCUSSION TITLE
Meal planner shows 8 days at a time, weeks start on the wrong days and overlap

### DISCUSSION THREAD

**Discussion by bverkron** [mealie-recipes/mealie#1780] [General]
Just migrated from v0.5 and it showed Sun-Sat and each week in the planner started on the same day. In v1 this week Tues-Tues, next week is Sun-Sun with overlap of the previous week (see screenshots). This makes planning a week at a time (day Sun-Sat) difficult. Can this be changed?

![5AC15A3E-7B4A-46F5-A486-41D379FCFD84](https://user-images.githubusercontent.com/19174503/198199264-d2781347-ed8a-4efa-930b-77a47f982d9b.png)

![38A7AA8A-69B7-4DB3-ABFF-82264291B438](https://user-images.githubusercontent.com/19174503/198199275-da06e650-2056-4294-beb0-77031c874615.png)

**Comment by kenske** [comment]
The ability to select a weekly or monthly view would be very helpful. The current UX pattern is not very intuitive.

**Comment by michael-genson** [comment]
This is now customizable: https://github.com/mealie-recipes/mealie/pull/3650
